resources/definitions/cartesio.def.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "cartesio",
+    "name": "Cartesio",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Scheepers",
+        "manufacturer": "Cartesio bv",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "has_machine_materials": true,
```

Comment:
I guess this is ready to merge?
There will be updates to the "extruder start code" as soon as the program matures.